src/app/Sidebar/SidebarUsers.js

```diff
@@ -74,13 +91,25 @@ export default class SidebarMessages extends Component {
     ));
 
     return (
-      <div className="SidebarMessages">
+      <div className="SidebarUsers">
         <ul className="tags">
+          <li className="search">
+            <div className="input-group">
+              <span className="input-group-addon"><Icon name="search" sm /></span>
+              <input
+                type="text"
+                className="form-control"
+                placeholder="Search"
+                value={search}
+                onChange={this.search}
+              />
+            </div>
+          </li>
           {unreadMessages.length ? (
             unreadMessages
           ) : null}
-          {contacts.length ? (
-            contacts
+          {users.length ? (
```

Comment:
Much cleaner to have the braces in the same row number to improve readability, also I prefer this style for single line if statements:
```
{ contacts.length
    ? ( contacts )
    : null
}